Dfe.Data.SearchPrototype/Infrastructure/Tests/CognitiveSearchServiceAdapterTests.cs

```diff
@@ -88,54 +87,18 @@ public async Task Search_SendsCorrectRequestToSearchService()
         searchOptionsPassedToSearchService?.Facets.Should().BeEquivalentTo(searchServiceAdapterRequest.Facets);
     }
 
```

Comment:
Missing test to check that the SearchOptionsBuilder is called correctly?

Reply:
So are you talking about adding verifications for everything we inject into the constructor, think this was a test you put in originally so did enough to correct based on the revised dependency?

Reply:
For a unit test I would  test that everything I pass into SearchAsync is passed on to the calls made from within the method, so I suppose in this case, I could check that every method of the builder class is called, or I could check that the _searchByKeywordService.SearchAsync method is called with all the right values .. which is almost done, except that not all of the searchOptions passed in are checked. 
If I comment out lines 83-85 in the CognitiveSearchAdapter::SearchAsync method, no tests fail, so the test that's called Search_SendsCorrectRequestToSearchService isn't checking that everything is sent